### ISSUE TITLE
Training details issues

### ISSUE THREAD

**Issue by DAUDafterDAUD** [Zhongping-Zhang/SGC_Net#1]
Amazing work! Can you tell me the GPU you are using and the training duration for the two stages？

**Comment by Zhongping-Zhang** [comment]
Hi, thank you for your interest in our work. The training process of **RoI prediction** is fairly fast. It just takes **a few minutes** to get results if you implement "train_clevr_triples2roi.py". Finetuning **Stable Diffusion** would be time-consuming, _e.g._, **3 hours per epoch** on VisualGenome (either A6000 or A40). _An alternative way is to directly apply pretrained ControlNet or SD on masked images_. Their results should still be fine, though not as good as we presented in the paper.
